control-plane-pki-operator/certificatesigningcontroller/certificatesigningcontroller_test.go

```diff
@@ -493,7 +532,7 @@ func TestSign(t *testing.T) {
 		MaxPathLen:            -1,
 	}
 
-	if d := cmp.Diff(*certs[0], want, diff.IgnoreUnset()); d != "" {
+	if d := cmp.Diff(*certs[0], want, IgnoreUnset()); d != "" {
```

Comment:
was this removed from the `diff` package?

Reply:
yes, it was. So copied the previous function over to our file